serverless/src/step_function/span-link.ts

```diff
@@ -113,12 +127,16 @@ function parseDefinitionObjectFromDefinitionString(
 function dumpDefinitionObjectAsDefinitionString(
   definitionObj: StateMachineDefinition,
   definitionFormat: StateMachineDefinitionFormat,
+  originalDefinitionString: DefinitionString,
```

Comment:
Need to pass this because `definitionObj` is only for the index-0 element of the array. Subsequent elements are untouched and are also needed here.